validation/custom_constraint.rst

```diff
@@ -65,6 +65,10 @@ The validator class is also simple, and only has one required method ``validate(
     {
         public function validate($value, Constraint $constraint)
         {
+            if (!$constraint instanceof ContainsAlphanumeric) {
```

Comment:
maybe `\assert($constraint instanceof ContainsAlphanumeric)` would be a better fit here?

Reply:
This is inline with how all core validators are implemented.